centos5-devtoolset2-gcc4/build_scripts/build.sh

```diff
@@ -9,10 +9,8 @@ CPYTHON_VERSIONS="3.6.4"
 
 # openssl version to build, with expected sha256 hash of .tar.gz
 # archive
-OPENSSL_ROOT=openssl-1.0.2o
-# Hash from https://www.openssl.org/source/openssl-1.0.2o.tar.gz.sha256
-# Matches hash at https://github.com/Homebrew/homebrew-core/blob/1766321103d9780f6e38d3ac7681b8fa42cdca86/Formula/openssl.rb#L11
-OPENSSL_HASH=ec3f5c9714ba0fd45cb4e087301eb1336c317e0d20b575a125050470e8089e4d
+OPENSSL_ROOT=openssl-1.0.2r
+OPENSSL_HASH=ae51d08bba8a83958e894946f15303ff894d75c2b8bbd44a852b64e3fe11d0d6
```

Comment:
Please, also update the comment after explicitly checking the hashes match.

Reply:
I agree. I removed the comment 30 seconds before your suggestion.